easybuild/easyconfigs/m/MEME/MEME-5.4.1-gompi-2021b.eb

```
@@ -0,0 +1,62 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'MEME'
+version = '5.4.1'
+
+homepage = 'https://meme-suite.org/meme/index.html'
+description = """The MEME Suite allows you to: * discover motifs using MEME, DREME (DNA only) or
+ GLAM2 on groups of related DNA or protein sequences, * search sequence databases with motifs using
+ MAST, FIMO, MCAST or GLAM2SCAN, * compare a motif to all motifs in a database of motifs, * associate
+ motifs with Gene Ontology terms via their putative target genes, and * analyse motif enrichment
+ using SpaMo or CentriMo."""
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+
+source_urls = ['https://%(namelower)s-suite.org/%(namelower)s/%(namelower)s-software/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['c07fb8afafa60fc5e84ca24493c82fa6f4bd1df1a2622102edbf86a1c30fd11f']
+
+dependencies = [
+    ('libxml2', '2.9.10'),
+    ('libxslt', '1.1.34'),
+    ('zlib', '1.2.11'),
+    ('Perl', '5.34.0'),
+    ('Python', '3.9.6'),
+    ('Ghostscript', '9.54.0'),
+    ('XML-Compile', '1.63'),
+]
+
+configopts = '--with-perl=${EBROOTPERL}/bin/perl --with-python=${EBROOTPYTHON}/bin/python '
+configopts += '--with-gs=${EBROOTGHOSTSCRIPT}/bin/gs '
+# config.log should indicate that all required/optional dependencies were found (see scripts/dependencies.pl)
+configopts += " && grep 'All required and optional Perl modules were found' config.log"
+
+pretestopts = "OMPI_MCA_rmaps_base_oversubscribe=1 "
+runtest = 'test'
+
+fix_perl_shebang_for = ['bin/*', 'libexec/meme-%(version)s/*']
+fix_python_shebang_for = ['bin/*', 'libexec/meme-%(version)s/*']
+
+sanity_check_paths = {
+    'files': ['bin/meme', 'bin/dreme', 'bin/meme-chip', 'libexec/meme-%(version)s/meme2meme'],
+    'dirs': ['lib'],
+}
+
+sanity_check_commands = [
+    "meme -h 2>&1 | grep 'Usage:'",
```

@SethosII this check fails for me because it needs to be invoked with mpirun in our clusters:
```suggestion
    "mpirun meme -h 2>&1 | grep 'Usage:'",
```